src/GoogleFit/FitOnFhir.GoogleFit.Tests/GoogleFitDataImporterTests.cs

```diff
@@ -142,6 +138,29 @@ public async Task GivenRefreshTokenThrowsTokenRefreshException_WhenImportIsCalle
                 Arg.Is<CancellationToken>(token => token == _cancellationToken));
         }
 
+        [Fact]
+        public async Task GivenAuthTokenResponseIsNull_WhenImportIsCalled_ImportOperationIsSkipped()
+        {
+            SetupMockSuccessReturns();
+            _googleFitTokensService.RefreshToken(Arg.Any<string>(), Arg.Any<CancellationToken>()).Returns(Task.FromResult<AuthTokensResponse>(null));
+
+            await _googleFitDataImporter.Import(_userId, _googleUserId, _cancellationToken);
+
+            await _googleFitClient.DidNotReceive().DataSourcesListRequest(Arg.Any<string>(), Arg.Any<CancellationToken>());
+            await _googleFitImportService.ProcessDatasetRequests(Arg.Any<GoogleFitUser>(), Arg.Any<IEnumerable<DataSource>>(), Arg.Any<AuthTokensResponse>(), Arg.Any<CancellationToken>());
```

Comment:
Isn't this missing the .DidNotReceive() qualifier?  There are also no checks to make sure that none of the `_googleFitUserTableRepository` calls were received, but perhaps that's not as important.

Reply:
It is. It's weird that this doesn't cause some kind of issue. The test passes just fine.